test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -260,46 +254,43 @@ public async Task InstallPackageToProjectsFromUI_PackageNamespace_WithSingleFeed
         }
 
         [StaFact]
-        public async Task InstallPackageFromUI_PackageNamespace_WithMultiFeed_Succeed()
+        public async Task InstallPackageFromUI_PC_PackageNamespace_WithMultiFeed_Succeed()
         {
             // Arrange
             EnsureVisualStudioHost();
             var solutionService = VisualStudio.Get<SolutionService>();
             string solutionDirectory = CommonUtility.CreateSolutionDirectory(Directory.GetCurrentDirectory());
-            solutionService.CreateEmptySolution("TestSolution", solutionDirectory);
-
-            var packageName = "Contoso.A";
-            var packageVersion1 = "1.0.0";
-
-            var externalRepositoryPath = Path.Combine(solutionService.ContainingDirectory, "ExternalRepository");
+            var externalRepositoryPath = Path.Combine(solutionDirectory, "ExternalRepository");
             Directory.CreateDirectory(externalRepositoryPath);
-            await CommonUtility.CreatePackageInSourceAsync(externalRepositoryPath, packageName, packageVersion1);
-
-            var privateRepositoryPath = Path.Combine(solutionService.ContainingDirectory, "PrivateRepository");
+            var privateRepositoryPath = Path.Combine(solutionDirectory, "PrivateRepository");
             Directory.CreateDirectory(privateRepositoryPath);
-            await CommonUtility.CreatePackageInSourceAsync(privateRepositoryPath, packageName, packageVersion1);
 
             // Create nuget.config with Package namespace filtering rules before project is created.
             CommonUtility.CreateConfigurationFile(Path.Combine(solutionDirectory, "NuGet.config"), $@"<?xml version=""1.0"" encoding=""utf-8""?>
 <configuration>
     <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
-    <add key=""ExternalRepository"" value=""{externalRepositoryPath}"" />
-    <add key=""PrivateRepository"" value=""{privateRepositoryPath}"" />
+    <add key=""encyclopaedia"" value=""{externalRepositoryPath}"" />
```

Comment:
Test "encyclopaedia" and "encyclopædia" sources used together doesn't crash VS PMUI setting.